src/metabase/pulse/markdown.clj

```diff
@@ -342,14 +399,16 @@
 
 (defmulti process-markdown
   "Converts a markdown string from a virtual card into a form that can be sent to a channel
-  (mrkdwn for Slack; HTML for email)."
+  (Slack's markup language, or HTML for email)."
   (fn [_markdown channel-type] channel-type))
 
-(defmethod process-markdown :mrkdwn
```

Comment:
I changed this from `:mrkdwn` to `:slack` and removed other references to `mrkdwn` because even though Slack's syntax is technically called "mrkdwn", it's a needlessly confusing name to use in our code